microservice/src/main/java/nearsoft/skytouch/microservice/controller/ProductController.java

```diff
@@ -0,0 +1,40 @@
+package nearsoft.skytouch.microservice.controller;
+
+import nearsoft.skytouch.common.model.Product;
+import nearsoft.skytouch.microservice.service.ProductService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+class ProductController {
+
+    private ProductService productService;
+
+    public ProductController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    @RequestMapping(value = "products", method = RequestMethod.GET)
+    private ResponseEntity<List<Product>> getProducts() {
+        List<Product> products = new ArrayList<>();
```

Comment:
do you need `products`?

Reply:
No i don't but i forgotto delete the controller, the microservice will not have a this class